requirements.txt

```diff
@@ -1 +1 @@
-pyparsing==2.0.2

```

Comment:
I think I intentionally used `==2.0.2` to make sure I required a pyparsing I'd tested with. But maybe it's unnecessarily strict?